### PR TITLE
SF-2211: modify job creation to accept json for the entire job creati

### PR DESCRIPTION
**Pull request by rklind**
…on block to support scheduling and personal_drive jobs

### CODE REVIEW

File: src/commands/jobs/add.ts [hunk 1]
```diff
@@ -52,21 +50,25 @@ export = {
...
+			let jobParametersInput = argv.jobInput ? await readJsonInput() : undefined;
+			if (!jobParametersInput && argv.jobFile) {
+				jobParametersInput = JSON.parse(fs.readFileSync(argv.jobFile, 'utf-8'));
 			}
-			job[job.kind] = options;
 
-			if (!options && !argv.optionsInput && job.kind === 'transfer' && argv.wizard) {
+			let jobDefaults: any = {
+				kind: 'transfer',
+				schedule: { mode: 'auto' }
```
Comment (recordrn) [inline src/commands/jobs/add.ts]: I would suggest defaulting the mode to manual

Comment (rklind) [reply src/commands/jobs/add.ts]: the default mode was already auto.  i didn't change that as part of this code change.  is there a reason we should change from our standard?

Comment (recordrn) [reply src/commands/jobs/add.ts]: from what I recall, the jobs created via CLI where all manual (so maybe that original default was not working). Most use cases for creating jobs via scripting will be to create jobs ahead of time, but not executing them.  If this was set to 'auto' as a default, I would most likely override it and set to manual for the standard use scripts.

**Comment by recordrn** [comment]
are we going to publish a new skysync-cli npm package?

**Comment by smbecker** [comment]
We generally only do that with the next software release.

**Pull request merged**